core/utils/poly-look/src/react/tabs/swipableTabs.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import Tab from "./tab.jsx";
+
+import "./tabs.css";
+
+const SwipableTabs = ({ children }) => {
+  const [activeTab, setActiveTab] = useState(children[0].props.label);
+
+  const onTabClick = (ev, newActiveTabId) => {
+    ev.preventDefault();
+    setActiveTab(newActiveTabId);
+  };
+
+  return (
+    <div className="tabs swipable-tabs">
+      <div className="tab-buttons">
+        {children.map((tab, index) => (
+          <button
+            key={index}
+            onClick={(ev) => onTabClick(ev, tab.props.label)}
+            className={
+              tab.props.label === activeTab ? "tab-button active" : "tab-button"
+            }
+          >
+            {tab.props.label}
+          </button>
+        ))}
+      </div>
```

Comment:
To reduce duplication for now (at least it looks duplicated at first glance), you could add a `TabButtons` component for this and use them both here and in `Tabs`, hm?